clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3605,7 +3605,8 @@ static void RenderSSPOptions(const Driver &D, const ToolChain &TC,
     StringRef Value = A->getValue();
     if (!EffectiveTriple.isX86() && !EffectiveTriple.isAArch64() &&
         !EffectiveTriple.isARM() && !EffectiveTriple.isThumb() &&
-        !EffectiveTriple.isRISCV())
+        !EffectiveTriple.isRISCV() && !EffectiveTriple.isPPC64() &&
+        !EffectiveTriple.isPPC32())
```

Comment:
Any reason not to use `EffectiveTriple.isPPC()` instead of both `EffectiveTriple.isPPC32()` and `EffectiveTriple.isPPC64()`?

Reply:
None at all -- I started adding just PPC64 support and realized that it would be easy to just do both 32 and 64 in the same series, but didn't go back and re-factor. Thanks for catching this. I've rebased on top of main so that the RISC-V changes aren't mixed into this PR.